tensorflow/core/transforms/remapper/pass.cc

```diff
@@ -91,18 +96,227 @@ class MatchMulSigmoid : public RewritePattern {
   OperationName sigmoid_name_;
 };
 
+namespace {
+class RewriterBase : public RewritePattern {
+ public:
+  RewriterBase(StringRef opName, OpPropertyHelper &helper,
+               PatternBenefit benefit = PatternBenefit(1))
+      : RewritePattern(opName, benefit, helper.getDialect()->getContext()),
+        helper_(helper),
+        dialect_(helper.getDialect()) {}
+  RewriterBase(MatchAnyOpTypeTag tag, OpPropertyHelper &helper,
+               PatternBenefit benefit = PatternBenefit(1))
+      : RewritePattern(tag, benefit, helper.getDialect()->getContext()),
+        helper_(helper),
+        dialect_(helper.getDialect()) {}
+
+ protected:
+  OpPropertyHelper &helper_;
+  TFGraphDialect *dialect_;
```

Comment:
Should be enough to get it from the helper_?

Reply:
Yes, removed this redundant member.